.github/workflows/ci.yml

```diff
@@ -24,21 +24,41 @@ jobs:
 
     - uses: leafo/gh-actions-luarocks@v4
 
+    - name: get foreman and run foreman install
+      uses: rojo-rbx/setup-foreman@v1
```

Comment:
Why are you using `rojo-rbx/setup-foreman@v1`
If I recall correctly, it's actually `Roblox/setup-foreman`

Reply:
great question! the foreman project was spun out into the community where it could be more consistently maintained.